docs/pipelines-in-pipelines.md

```diff
@@ -0,0 +1,106 @@
+<!--
+---
+linkTitle: "Pipelines in Pipelines"
+weight: 406
+---
+-->
+
+# Pipelines in Pipelines
+> 🌱 Pipelines in Pipelines is an alpha feature. The enable-api-fields feature flag must be set to "alpha" to specify PipelineRef or PipelineSpec in a PipelineTask.
```

Comment:
Please add a note that this feature is in a preview mode and not implemented yet.